File: app/src/main/java/com/sajibmamun/quizo/MainActivity.java
package com.sajibmamun.quizo;

import androidx.appcompat.app.AppCompatActivity;

import android.content.Intent;
import android.os.Bundle;
import android.view.View;
import android.widget.Button;

public class MainActivity extends AppCompatActivity {
    Button startQuizBtn,bookmarkBtn;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        startQuizBtn=findViewById(R.id.StartQuizBtn);
        bookmarkBtn=findViewById(R.id.BookMarkBtnid);

        startQuizBtn.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {
                Intent intent=new Intent(MainActivity.this,CatagoriesActivity.class);
                startActivity(intent);

            }
        });
    }
}

File: app/src/main/java/com/example/quizo/MainActivity.kt
package com.example.quizo

import android.content.Intent
import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import android.view.View
import android.widget.Toast
import com.cncoderx.wheelview.OnWheelChangedListener
import com.example.quizo.databinding.ActivityMainBinding

class MainActivity : AppCompatActivity() {


    var topic:String?="Java"
    override fun onCreate(savedInstanceState: Bundle?) {
        var binding:ActivityMainBinding
        super.onCreate(savedInstanceState)
        binding= ActivityMainBinding.inflate(layoutInflater)
        setContentView(binding.root)



        binding.topicwheel.onWheelChangedListener =
            OnWheelChangedListener { view, _, newIndex ->
                val text = view.getItem(newIndex)
                topic=text.toString()
            }


        binding.playbtnid.setOnClickListener(View.OnClickListener {



                var intent1=Intent(applicationContext,QuestionClassActivity::class.java)
                intent1.putExtra("subject",topic)
                startActivity(intent1)


        })
    }
}